nservicebus/sqlserver/deployment-options.md

```diff
@@ -23,6 +23,21 @@ NOTE: To properly identify the chosen deployment option all queues that the endp
 The transport will route messages to destination based on the configuration. If no specific configuration has been provided for a particular destination, the transport assumes the destination has the same configuration as the sending endpoint (i.e. identical schema, catalog and instance name). If the destination has a different configuration and it hasn't been provided, then exception will be thrown when sending a message, because the transport wouldn't be able to connect to the destination queue.
 
 
+### Example
+
+The following table shows sample possible fully qualified queue table names for different deployment options:
```

Comment:
>The following table shows sample possible fully qualified queue table names for different deployment options:

How about:
The following table shows the resulting queue table name for each deployment option. In these examples, the SQL server name is assumed to be `sql_server_01`.